pom.xml

```
@@ -222,6 +222,7 @@
     <maven-site-plugin.version>3.21.0</maven-site-plugin.version>
     <maven-source-plugin.version>3.3.1</maven-source-plugin.version>
     <maven-surefire-plugin.version>3.5.4</maven-surefire-plugin.version>
+    <sonar-maven-plugin.version>5.2.0.4988</sonar-maven-plugin.version>
```

```suggestion
    <sonar-maven-plugin.version>5.3.0.6276</sonar-maven-plugin.version>
```

There's a new release with a few bugfixes we might want to pin: https://github.com/SonarSource/sonar-scanner-maven/releases/tag/5.3.0.6276

Otherwise looks good, assuming you'll check the Sonar scan outcome and see that it works as intended 👍